device-api/src/device.rs

```diff
@@ -376,7 +383,8 @@ impl DeviceInfo {
         Ok(node)
     }
 
-    pub fn get_performance_counter(&self, file: &DeviceFile) -> DeviceResult<PerformanceCounter> {
+    #[allow(dead_code)]
+    fn get_performance_counter(&self, file: &DeviceFile) -> DeviceResult<PerformanceCounter> {
```

Comment:
This code is used in PR 81. I think we can keep it public and remove the dead_code as well.